source/screens/caseScreens/CaseSummary.js

```diff
@@ -479,18 +479,28 @@ const CaseSummary = (props) => {
                         )}
 
                         <Card.DetailsTitle type="h5">Reducering</Card.DetailsTitle>
-
                         {calculation?.reductions?.reduction ? (
                           <>
-                            <Card.CalculationRow>
-                              <Card.CalculationRowCell>
-                                <Text>{calculation.reductions.reduction}</Text>
-                              </Card.CalculationRowCell>
-                              <Card.CalculationRowCell />
-                              <Card.CalculationRowCell>
-                                <Text>{formatAmount(calculation.reductions.reduction.amount)}</Text>
-                              </Card.CalculationRowCell>
-                            </Card.CalculationRow>
+                            {convertDataToArray(calculation?.reductions?.reduction).map(
```

Comment:
There are a few `reduction?.xxx` which implies that `reduction` can sometimes be null/invalid, however `reduction.amount` (line 499) does not do this, meaning it would crash there regardless.

If `reduction` is expected to ever be null/invalid I would suggest performing a `filter` before `map`. Something like:

```
{convertDataToArray(calculation?.reductions?.reduction)
  .filter((r) => r)
  .map((reduction, index) => (
    ...
  ))}
```

If it's not supposed to ever be null I would remove the `?` for consistency.